src/components/Post/Tabs/PostTimeline/TimelineContainer.tsx

```diff
@@ -25,87 +41,135 @@ interface ITimelineContainerProps {
 		hash: string;
 		statuses: BlockStatus[];
 		type?: string;
-	}
+	};
 }
 
 function sortfunc(a: BlockStatus, b: BlockStatus) {
 	return a.block - b.block;
 }
 
 const TimelineContainer: React.FC<ITimelineContainerProps> = (props) => {
+	const [isCollapsed, setIsCollapsed] = useState(false);
 	const { timeline } = props;
+	const { postData: { postType } } = usePostDataContext();
+	const PostType = postType.replace(/(^|_)([a-z])/g, (_, __, c) => c.toUpperCase()).replace(/s$/, '');
+	let activeColor;
+	PostType === timeline.type ?  activeColor = '#485F7D' : activeColor = '#E5007A';
 	const { network } = useNetworkContext();
 	if (!timeline) return null;
 	const { statuses, type } = timeline;
 
 	if (statuses.length === 0) return null;
+	const minHeight = statuses.length * 50;
 
-	const StatusDiv = ({ status } : { status: string }) => {
-		return (
-			<div className='flex items-center absolute -top-3.5 justify-center'>
-				<StatusTag colorInverted={false} status={status} type={type} />
-			</div>
-		);
+	const toggleCollapse = () => {
+		setIsCollapsed(!isCollapsed);
 	};
 
+	let displayIconActive;
+	let displayIconUnactive;
+	if(getStatus(String(type)) == 'Referendum'){
+		displayIconActive = <DemocracyReferendaIcon className="-ml-[6px] mr-3 mt-2"/>;
+		displayIconUnactive = <DemocracyReferendaGreyIcon className="-ml-[6px] mr-3 mt-2" />;
+	}
+	else if(getStatus(String(type)) == 'Treasury Proposal'){
+		displayIconActive = <TreasuryProposalIcon className="-ml-[6px] mr-3 mt-2"/>;
+		displayIconUnactive = <TreasuryProposalGreyIcon className="-ml-[6px] mr-3 mt-2" />;
+	}
+	else if(getStatus(String(type)) == 'Motion'){
+		displayIconActive = <MotionIcon className="-ml-[6px] mr-3 mt-2"/>;
+		displayIconUnactive = <MotionIconGrey className="-ml-[6px] mr-3 mt-2" />;
+	}
+	else if(getStatus(String(type)) == 'Tech Committee Proposal'){
+		displayIconActive = <TechCommProposalIcon className="-ml-[6px] mr-3 mt-2"/>;
+		displayIconUnactive = <TechCommProposalIconGrey className="-ml-[6px] mr-3 mt-2" />;
+	}
+	else if(getStatus(String(type)) == 'Democracy Proposal'){
+		displayIconActive = <DemocracyProposalIcon className="-ml-[6px] mr-3 mt-2"/>;
+		displayIconUnactive = <DemocracyProposalIconGrey className="-ml-[6px] mr-3 mt-2" />;
+	}
+	else if(getStatus(String(type)) == 'Child Bounty'){
+		displayIconActive = <ChildBountyIcon className="-ml-[6px] mr-3 mt-2"/>;
+		displayIconUnactive = <ChildBountyIconGrey className="-ml-[6px] mr-3 mt-2" />;
+	}
+	else {
+		displayIconActive = <DiscussionIcon className="-ml-[6px] mr-3 mt-2"/>;
+		displayIconUnactive = <DiscussionIconGrey className="-ml-[6px] mr-3 mt-2" />;
+	}
 	const url = getBlockLink(network);
 
-	const TimelineItems = (isMobile:boolean) => {
-
-		return (
-			<section className={`flex-1 flex ${isMobile? 'flex-col items-start gap-y-20 py-20': 'items-center'}`}>
+	const Timeline = () => {
```

Comment:
did we create a ui for which in responsive and web, mobile friendly?

Reply:
yes